scripts/xbstrap

```diff
@@ -658,6 +665,44 @@ class TargetPackage(RequirementsMixin):
 		path = os.path.join(self._cfg.sysroot_dir, 'etc', 'xbstrap', self.name + '.installed')
 		touch(path)
 
+class RunPkgTarget(RequirementsMixin):
```

Comment:
The word `Target` in `RunPkgTarget` might be slightly confusing (since we already use "target" to refer to the target OS). However, I don't have any better suggestion right now.

Reply:
I chose it based on `make` terminology, but obviously this can be changed to something else. I just cannot come up with any better term right now.

Reply:
What about `PackageRunTask` (with `RunTask` refering to the command that you run)?

Reply:
Addressed in 6504f41.